docs/ja/front/component/ma-table.md

```
@@ -1,49 +1,70 @@
+| `showOverflowTooltip` | <el-tooltip content="内容が長すぎて隠されている場合に tooltip を表示">`マウスオーバーで表示`</el-tooltip>       | `boolean`                                                                                       | `false`  | 1.0.0 |
+| `adaption` | 高さを自動調整するか                                                                | `boolean`                                                                                       | `false`  | 1.0.0 |
 | `adaptionOffsetBottom` | 下部からのオフセット量                                                               | `number`                                                                                        | `70`     | 1.0.0 |
 | `showPagination` | ページネーションを表示するか                                                               | `boolean`                                                                                        | `true`     | 1.0.0 |
-| `pagination` | Elページネーションのネイティブ属性、イベント                                                           | [el-pageinationドキュメント](https://element-plus.org/zh-CN/component/pagination.html#%E5%B1%9E%E6%80%A7) | -        | 1.0.0 |
-| `on`      | `el-table` テーブルイベントの集合                                                     | <el-tooltip content="Object: { onSelect: (args) => {}, .... }">`マウスオーバーで表示`</el-tooltip>            | -        | 1.0.0 |
+| `pagination` | Elページネーションのネイティブ属性、イベント                                                           | [el-pageinationドキュメント](https://element-plus.org/ja/component/pagination.html#%E5%B1%9E%E6%80%A7) | -        | 1.0.0 |
+| `on`      | `el-table` テーブルイベントコレクション                                                     | <el-tooltip content="Object: { onSelect: (args) => {}, .... }">`マウスオーバーで表示`</el-tooltip>            | -        | 1.0.0 |
 
 
 #### LoadingConfig説明
 | パラメータ        | 説明      | タイプ   | デフォルト値 | バージョン    |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)